sklearn/model_selection/_validation.py

```diff
@@ -476,7 +476,7 @@ def _fit_and_score(estimator, X, y, scorer, train, test, verbose,
             msg = ''
         else:
             msg = '%s' % (', '.join('%s=%s' % (k, v)
-                          for k, v in parameters.items()))
+                                    for k, v in parameters.items()))
```

Comment:
Please avoid unrelated changes (other places as well)

Reply:
Also accidental change, reverted back in ea0cc71